changelog.md

```diff
@@ -2,6 +2,15 @@
 
 ## UNRELEASED
 
+**Breaking:**
+- react-native-bpk-component-text-input:
+  - Now supports Android.
+  - A new design, see https://backpack.github.io/components/native/text-input.
```

Comment:
Can we change this to "A new floating text label design" with no link? What happens when we change the style again in the future? Hypothetical I know, but the content at that url is dynamic so it shouldnt be in the changelog.